llm_studio/src/utils/modeling_utils.py

```diff
@@ -840,6 +839,13 @@ def create_nlp_backbone(cfg, model_class=AutoModel) -> Any:
                     "likely lead to unstable training without adapters."
                 )
 
+    for name, param in backbone.named_parameters():
+        # freeze base model's layers
+        if any(freeze_layer in name for freeze_layer in cfg.training.freeze_layers):
+            if cfg.environment._local_rank == 0:
+                logger.info(f"Freezing layer: {name}")
+            param.requires_grad = False
```

Comment:
Let's catch the cases here where LoRA is enabled and print a warning that freezing is ignored.

Reply:
I will just move it into the correct if without warning as this setting should anyways only be used without lora and docs should be clear there.